src/content/docs/en/guides/cms/sitecore.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: Sitecore Experience Manager & Astro
+description: Add content to your project using Sitecore as your CMS.
+type: cms
+stub: true
+service: Sitecore XM
+i18nReady: false
+---
+
+[Sitecore Experience Manager™ (XM)](https://www.sitecore.com/products/experience-manager) is a powerful and intuitive CMS for the simplified creation and management of experiences across channels and devices.
+
+## Getting started
+
+1. Create a Sitecore Headless website following official [documentation](https://doc.sitecore.com/xp/en/developers/sxa/103/sitecore-experience-accelerator/create-a-headless-tenant-and-site.html).
+2. Run the project initialization command in your terminal: 
+  ```shell
+  npx @astro-sitecore-jss/create-astro-sitecore-jss@latest
+  ```
+3. Follow the instructions in the terminal to create your project.
+4. Start Astro website:
+  ```shell
+  npm run dev
+  ```
+
+## Community Resources
+
+- [Sitecore JavaScript Software Development Kit for Astro](https://github.com/exdst/jss-astro-public) on GitHub.
+- [Introduction to Sitecore with Astro](https://exdst.com/posts/20231002-sitecore-astro).
+- [Starting Your First Sitecore Astro Project](https://exdst.com/posts/20240103-first-sitecore-astro-project).
+- [Creating Your First Sitecore Astro Rendering](https://exdst.com/posts/20240104-sitecore-astro-rendering).
```

Comment:
```suggestion
```
This page is linked to from the Starting your first Project article, so it's fine just to have the Getting Started one here.